codbex-payment-order-ext/generate/SalesOrderPayment/api/GenerateSalesOrderPaymentService.ts

```diff
@@ -1,4 +1,70 @@
+import { SalesOrderRepository as SalesOrderDao } from "../../../../codbex-orders/gen/codbex-orders/dao/SalesOrder/SalesOrderRepository";
+import { CustomerPaymentRepository as CustomerPaymentDao } from "../../../../codbex-payments/gen/codbex-payments/dao/CustomerPayment/CustomerPaymentRepository";
+import { SalesOrderPaymentRepository as SalesOrderPaymentDao } from "../../../../codbex-orders/gen/codbex-orders/dao/SalesOrder/SalesOrderPaymentRepository";
```

Comment:
You could try to use absolute imports instead of relative:
```ts
import { SalesOrderRepository as SalesOrderDao } from "codbex-orders/gen/codbex-orders/dao/SalesOrder/SalesOrderRepository";
```

Reply:
I understood and fix that